src/e2e/specs/dashboard.spec.ts

```diff
@@ -642,3 +718,42 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Navigation`, (
     );
   });
 });
+
+test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Data Breaches`, () => {
+  test.beforeEach(async ({ landingPage, page, authPage }) => {
+    await forceLoginAs(
+      "joe@mailinator.com",
+      "TestPass1234",
```

Comment:
Interesting... How does this work? If you reuse the same user for every test/run, isn't the profile contaminated?
Do we ever destroy this user?
Do we have steps for if we need a new fake test acct in the future? (just pre-create an account on dev/stage/production with the exact same usernames/passwords so the tests will run everywhere?)

Reply:
commented in the other PR, it's not a good security practice to list out credentials in plain text

Reply:
This user just has some nice properties which are convenient for this test. And i dont destroy the user. What do you mean by contaminated btw?